### DISCUSSION TITLE
Seeking Guidance to receive mag sensor data from the prototype deck and control Yaw attitude

### DISCUSSION THREAD

**Discussion by minisol0175** [bitcraze/discussions#864] [Q&A]
Hello, I want to design the system that the prototype deck attached with additional magnetometer work for Yaw angle estimation.

So I think I should know about usage of the prototype deck. I read the guidance as follows:
https://www.hackster.io/evoggy/flying-traffic-light-8be87d

But Now the firmware branch of "master" is changed much.

Can you help me the usage of the prototype deck with the firmware branch of "master" or over "2023.02"?

**Comment by knmcguire** [comment]
Ah yes it now changed because we use Kbuild.

I think these instructions can help you a bit further: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/howto/. Perhaps combine it with the hackster IO instructions.
'
We should probably start converting the hackster.io project into our own documentation or else things get easily outdated 😅
